package.json

```diff
@@ -157,9 +153,9 @@
     "postcss-svg": "^3.0.0",
     "prop-types": "^15.6.0",
     "raf": "^3.4.0",
-    "react": "^16.8.0 || ^17.0.0-rc.1",
+    "react": "^17.0.2",
     "react-axe": "^3.0.2",
-    "react-dom": "^16.8.0 || ^17.0.0-rc.1",
+    "react-dom": "^17.0.2",
     "react-overlays": "0.8.3",
     "react-test-renderer": "^16.9.0",
```

Comment:
should we update this as well?
i'm not sure why it wasn't included in the install-17 script

Reply:
Do we even use it? I don't see any references. And the tests seem to pass?